imap_processing/tests/codice/test_codice_l1a.py

```diff
@@ -237,6 +241,53 @@ def test_l1a_validate_data_arrays(test_l1a_data: xr.Dataset, index):
         pytest.xfail(f"Still need to implement validation for {descriptor}")
 
 
+def test_l1a_validate_hskp_data(test_l1a_data):
+    """Tests that the L1a housekeeping data is valid"""
+
+    # Housekeeping data is the 2nd element in the list of test products
+    hskp_data = test_l1a_data[2]
+    validation_hskp_filepath = VALIDATION_DATA[2]
+
+    # Load the validation housekeeping data
+    validation_hskp_data = load_cdf(validation_hskp_filepath)
+
+    # These variables are present in the decommed test data, but not present in
+    # the validation data
+    # TODO: Ask Joey if these can be removed from the L1a housekeeping CDFs
+    exclude_variables = [
+        "spare_1",
+        "spare_2",
+        "spare_3",
+        "spare_4",
+        "spare_5",
+        "spare_6",
+        "spare_62",
+        "spare_68",
+    ]
+
+    # These variables are named differently between the decommed test_data and
```

Comment:
Would it make sense to do a mapping for now instead of ignoring these?

Reply:
Actually I was mistaken here. These variables are simply not present in the validation data. I updated the comment here to reflect this and will ask Joey about it.